site/_includes/explanations/tenants-namespaces.md

```diff
@@ -0,0 +1,42 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+Pulsar was designed from the ground up to be a {% popover multi-tenant %} system. In Pulsar, {% popover tenants %} are the highest administrative unit within a Pulsar {% popover instance %}.
+
+### Tenants
```

Comment:
I think we should also mention that there's a `public` tenant pre-created in the system and a `public/default` namespace.